tests/test_routes.py

```diff
@@ -176,3 +176,11 @@ def test_create_order_bad_status(self):
         test_order["status"] = "created"  # wrong value
         response = self.app.post(BASE_URL, json=test_order)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_list_orders(self):
+        """ It should list orders"""
+        self._create_orders(5)
+        response = self.app.get(BASE_URL)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        data = response.get_json()
+        self.assertEqual(len(data), 5)
```

Comment:
Please run `make lint` to fix any lint issues introduced with your code. For instance, you have not added a new line to the end of the file.

Reply:
Running linting...
flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
0
flake8 . --count --max-complexity=10 --max-line-length=127 --statistics
./service/routes.py:7:1: F401 'flask.Flask' imported but unused
./tests/test_routes.py:11:1: F401 'unittest.mock.MagicMock' imported but unused
./tests/test_routes.py:11:1: F401 'unittest.mock.patch' imported but unused
3     F401 'flask.Flask' imported but unused
3
make: *** [Makefile:31: lint] Error 1

Do you want me to get rid of the unused?

Reply:
I think leave the unused imports for now because they may be used later.

Reply:
I removed them to see further reports on the formatting. We can always add it if we need it in the future.